libraries/ui-library/src/components/six-datepicker/six-date-formats.tsx

```diff
@@ -10,6 +10,11 @@ export enum SixDateFormats {
   DDMMYY_SLASH = 'dd/mm/yy',
   YYMMDD_SLASH = 'yy/mm/dd',
 
+  DDMMYYY_DOT_TIME_MINUTES = 'dd.mm.yyyy hh:MM',
+  YYYYMMDD_DASH_TIME_MINUTES = 'yyyy-mm-dd hh:MM',
```

Comment:
Why do we need all those constants/enums? As far as I can see, we never check if the user uses one of them. It might be sufficient to simply document the valid values and use a string as the type for the datepicker's date format.

Reply:
That is a good question. @johann-latocha what was the initial behind this, can you clarify? :)